### DISCUSSION TITLE
Currency Symbol - Feature Request

### DISCUSSION THREAD

**Discussion by cello8619** [teslamate-org/teslamate#972] [Ideas]
Hello, for all of the cost fields can we have the ability to add the currency symbol.  I know it is possible in grafana but the existing dashboards will not save, I can copy the dashboards and re-do but then the links from the TeslaMate main page would not bring me to the updated dashboards.  I was thinking it would be nice in the settings screen in TeslaMate to choose either no currency symbol or be able to pick from a list and then all the dashboards would reflect the correct currency.

**Comment by 3isenHeiM** [comment]
Indeed, this would be nice to have them defined site-wise.

In the settings webpage, we could have another field with the currency

![image](https://user-images.githubusercontent.com/26417172/220064752-a229e4f5-d5d4-4409-bbc4-6d08f9978fb9.png)

Then it could populate the global_settings file https://github.com/adriankumpf/teslamate/blob/master/lib/teslamate/settings/global_settings.ex
